Microsoft.HealthVault/Connection/SessionCredentialClientBase.cs

```diff
@@ -59,6 +59,8 @@ private SessionCredential GetSessionCredential(HealthServiceResponseData respons
             }
 
             SessionCredential sessionCredential = this.GetAuthenticationToken(responseData.InfoNavigator);
+            // Based on access token default lifetime: https://docs.microsoft.com/en-us/azure/active-directory/active-directory-configurable-token-lifetimes
```

Comment:
this is not active directory thing and is custom to healthvault .. it should reflect what the server is sending .. which should be 24 hours ..  

---
In reply to: [115806561](https://github.com/Microsoft/healthvault-dotnetstandard-sdk/pull/27#discussion_r115806561) [](ancestors = 115806561,115805648,115798063)